flang/include/flang/Lower/Support/BoxValue.h

```diff
@@ -404,6 +358,9 @@ class MutableBoxValue : public AbstractIrBox {
   MutableProperties mutableProperties;
 };
 
+/// Used for triple notation (array slices)
+using RangeBoxValue = std::tuple<mlir::Value, mlir::Value, mlir::Value>;
```

Comment:
Is this still used? (I thought we had removed it a while back.)

Reply:
Yes, it is still used around subscript-triplet lowering here:
https://github.com/flang-compiler/f18-llvm-project/blob/491e6f11e96177ddaa10e164d8866786c3db6db5/flang/lib/Lower/ConvertExpr.cpp#L995

It is a very localized use, so the definition could also be moved to `ConvertExpr.cpp` to avoid being exported in a public header.